src/main/kotlin/bakalarka/elearningplatform/service/EnrollmentService.kt

```diff
@@ -0,0 +1,58 @@
+package bakalarka.elearningplatform.service
+
+import bakalarka.elearningplatform.db.EnrollmentRepository
+import bakalarka.elearningplatform.model.Enrollment
+import bakalarka.elearningplatform.request.AddEnrollmentRequest
+import bakalarka.elearningplatform.request.UpdateProgressRequest
+import org.springframework.stereotype.Service
+
+@Service
+class EnrollmentService(
+    var enrollmentRepository: EnrollmentRepository,
+    var courseService: CourseService
+) {
+
+    fun findByUserId(): MutableSet<Enrollment> {
+        val id = UserService.getUserId()
+        return enrollmentRepository.findByUserID(id)
```

Comment:
Asi tiez - orElseThrow.

Reply:
Tu sa ocakava empty set na FE, v pripade ze nema ziadne enrollmenty

Reply:
Jaj okej. Pokial to necrashne na prazdnom, tak je to ok :+1:. Nevsimol som si, ze je to vracia nejaku collection, my bad.